src/youtube-player/README.md

```diff
@@ -0,0 +1,124 @@
+# Angular YouTube Player component
+
+This component provides a simple angular wrapper around the embed [YouTube player API](https://developers.google.com/youtube/iframe_api_reference). File any bugs against the [angular/components repo](https://github.com/angular/components/issues).
+
+## Installation
+
+To install, run `npm install @angular/youtube-player`.
+
+## Usage
+
+Follow the following instructions for setting up the YouTube player component:
+
+- First, follow the [instructions for installing the API script](https://developers.google.com/youtube/iframe_api_reference#Getting_Started).
+- Then make sure the API is available before bootstraping the YouTube Player component.
+- Provide find the video id by extracting it from the video URL.
+
+## Example
+
+If you're video is found at https://www.youtube.com/watch?v=PRQCAL_RMVo, then your video id is `PRQCAL_RMVo`.
+
+```typescript
+// example-module.ts
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {YouTubePlayerModule} from '@angular/youtube-player';
+import {YoutubePlayerExample} from './example-component';
+
+@NgModule({
+  imports: [
+    // Needed for ngIf directive.
+    CommonModule,
+    YouTubePlayerModule,
+  ],
+  declarations: [YoutubePlayerExample],
+})
+export class YoutubePlayerExampleModule {
+}
+
+// example-component.ts
+@Component({
+  moduleId: module.id,
+  template: '<youtube-player videoId="PRQCAL_RMVo" *ngIf="apiReady" />',
+  selector: 'youtube-player-example',
+})
+class YoutubePlayerExample implements OnInit {
+  /** Field preventing the YouTube player from bootstrapping until the api is ready. */
+  apiReady = false;
+
+  constructor(private _ref: ChangeDetectorRef) {}
+
+  onInit() {
+    // This code loads the IFrame Player API code asynchronously, according to the instructions at
+    // https://developers.google.com/youtube/iframe_api_reference#Getting_Started
+    var tag = document.createElement('script');
+
+    tag.src = "https://www.youtube.com/iframe_api";
+    var firstScriptTag = document.getElementsByTagName('script')[0];
+    firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+
+    // This function is called by the YouTube iframe api once the script is ready.
+    window.onYouTubeIframeAPIReady = () {
+      // Allow the YouTube player to bootstrap.
+      this.apiReady = true;
+
+      // Notify angular's change detection mechanism that something has changed.
+      this._ref.detectChanges();
+    };
+  }
+}
+
+```
+
+## API
+
+### Inputs
```

Comment:
We shouldn't document the inputs, outputs and methods here, because they'll go out of sync. We have some tooling that will pick up the documentation automatically from the source code.

Reply:
For now, though, we don't have a place to publish the generated docs. We could just link to the source, though, since the JsDoc is pretty readable (especially for these members)